code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -31,16 +31,16 @@
 	return TRUE
 
 /obj/item/projectile/bullet/a84mm_he
-	name ="\improper HE missile"
-	desc = "Boom."

```

Comment:
funny description gone again. Tupina, why do you ruin the only thing I enjoy playing anymore (nukie)